doc/faq.rst

```diff
@@ -22,25 +22,39 @@ need to install the ``virtualenv`` package, and then create a new virtual enviro
     virtualenv --python=python3 ~/eos
 
 
-The new virtual environment will now be activated.
+The new virtual environment will now be **activated**.
+That means, the `python` (and `pip`) command will now execute the local binary instead of a system-wide one.
 At any later point, you can (re-)activate the virtual environment by running the following command:
 
 ::
 
-    ~/eos/bin/activate
+    source ~/eos/bin/activate
 
-After activation, you can use install a binary distribution of EOS using ``pip3 install eoshep``.
+
+While activated, you can **install** a binary distribution of EOS using ``pip install eoshep``.
 If you plan to install EOS from source, as discussed in the :ref:`installation section<installation-from-source>`, you must set the prefix
 to match the path to your virtual environment, here: ``~/eos``.
 
-To deactivate the virtual environment, run
+You most likely want to install **jupyter and notebook**.
+To be able to access the virtual environment from the notebook interface, you also need to *install it* (details `here <https://ipython.readthedocs.io/en/stable/install/kernel_install.html#kernels-for-different-environments>`_).
```

Comment:
[...], you also need to make it known to jupyter (i.e., to *install it*; details `here ...`).

Would be nice explain why and how this install is different than the other. "make it known" helps?

Reply:
I stick to that nomenclature deliberately: we want to keep it as short as possible, the details are available in their doc. And I don't know precisely what's happening either; therefore this is safer.

Could I convince you? We could add quotation marks instead of the cursive to avoid confusion: "install it as a kernel".

Reply:
Fair enough, they use that nomenclature in their command. Keep as it is!